### DISCUSSION TITLE
How to use telescope-fzf-native?

### DISCUSSION THREAD

**Discussion by baraahomsi** [lumen-oss/rocks.nvim#546] [Q&A]
Luarocks documentation doesn't help much; still don't know how to get libfzf.so (compiled by telescope-fzf-native) in the _extensions folder.

**Comment by mrcjkb** [comment]
There's an open PR:

- https://github.com/nvim-telescope/telescope-fzf-native.nvim/pull/127

Until then, you have to install it with rocks-git.nvim.
